pkg/cmd/status.go

```diff
@@ -65,81 +65,125 @@ func getData(objType string, context string, namespace string, tenant string) (s
+		if err != nil {
+			return errors.Wrapf(err, "Prompt failed")
+		}
+
 		kubeconfig := utils.GetKubeconfig()
-		target, err := kubernetes.ParseSubject(args[0])
+		target, err := kubernetes.ParseSubject(name)
 		if err != nil {
 			pterm.Error.Println(err, "Its not a valid Summonplatform or Microservice")
 			os.Exit(1)
 		}
 
-		kubeObj := kubernetes.GetAppropriateObjectWithContext(*kubeconfig, args[0], target, inCluster)
+		kubeObj := kubernetes.GetAppropriateObjectWithContext(*kubeconfig, name, target, inCluster)
 		if reflect.DeepEqual(kubeObj, kubernetes.Kubeobject{}) {
-			pterm.Error.Printf("No instance found %s\n", args[0])
+			pterm.Error.Printf("No instance found %s\n", name)
 			os.Exit(1)
 		}
 
-		sData, err := getData("summon", kubeObj.Context.Cluster, target.Namespace, args[0])
-		if err != nil {
-			return err
-		}
-		dData, err := getData("deployment", kubeObj.Context.Cluster, target.Namespace, args[0])
-		if err != nil {
-			return err
+		var sData, dData, pData string
+		if statusType == "Summon Platform Status" {
+			sData, err = getData("summon", kubeObj.Context.Cluster, target.Namespace, name)
+			if err != nil {
+				return err
+			}
+			dData, err = getData("deployment", kubeObj.Context.Cluster, target.Namespace, name)
+			if err != nil {
+				return err
+			}
+		} else {
+			pData, err = getData("postgresdump", kubeObj.Context.Cluster, target.Namespace, name)
+			if err != nil {
+				return err
+			}
 		}
-		if follow {
+		followStatus, _ := cmd.Flags().GetBool("follow")
+		if followStatus {
 			area, _ := pterm.DefaultArea.WithRemoveWhenDone().Start()
 
 			for {
-				area.Update(sData, "\n", dData)
 				p := pterm.DefaultProgressbar.WithTotal(2)
 				p.ShowElapsedTime = false
 				p.RemoveWhenDone = true
 				_, _ = p.Start()
-				p.Title = "Fetching data"
-
-				// Calling it at end of for loop since we made these calls right before.
-				sData, err = getData("summon", kubeObj.Context.Cluster, target.Namespace, args[0])
-				if err != nil {
-					return err
+				if statusType == "Summon Platform Status" {
+					area.Update(sData, "\n", dData)
+					p.Title = "Fetching data"
+					sData, err = getData("summon", kubeObj.Context.Cluster, target.Namespace, name)
+					if err != nil {
+						return err
+					}
+					p.Increment()
+
+					dData, err = getData("deployment", kubeObj.Context.Cluster, target.Namespace, name)
+					if err != nil {
+						return err
+					}
+					p.Increment()
+				} else {
+					area.Update(pData)
+					pData, err = getData("posgtresdump", kubeObj.Context.Cluster, target.Namespace, name)
+					if err != nil {
+						return err
+					}
 				}
-				p.Increment()
-
-				dData, err = getData("deployment", kubeObj.Context.Cluster, target.Namespace, args[0])
-				if err != nil {
-					return err
-				}
-				p.Increment()
 				_, _ = p.Stop()
 				_ = area.Stop()
-
 			}
 		} else {
-			pterm.Success.Printf(sData + "\n" + dData + "\n")
+			if statusType == "summonplatform" {
```

Comment:
This will never execute because we don't have status type as summonplatform anymore